LoopKitUI/UIColor.swift

```diff
@@ -15,6 +15,10 @@ private func BundleColor(_ name: String, compatibleWith traitCollection: UITrait
 }
 
 extension UIColor {
+    @nonobjc static let lightenedInsulin = BundleColor("Lightened Insulin") ?? systemOrange
+    
+    @nonobjc static let darkenedInsulin = BundleColor("Darkened Insulin") ?? systemOrange
+    
```

Comment:
Colors for override percentage gauge bar

Reply:
Colors used in LoopKit should be provided by the app layer in some way. Passing colors into the view constructor, or assigning attributes on the view, or setting up Environment attributes for SwiftUI views.